### ISSUE TITLE
Uncaught Error: Repository has been destroyed

### ISSUE THREAD

**Issue by AdamSkaidon** [imdone/imdone-atom#59]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.3.1
**System**: Mac OS X 10.11.2
**Thrown From**: [imdone-atom](https://github.com/imdone/imdone-atom) package, v1.3.12
### Stack Trace

Uncaught Error: Repository has been destroyed

```
At /Users/Adam/.atom/packages/imdone-atom/node_modules/imdone-core/lib/mixins/repo-watched-fs-store.js:0

Error: Repository has been destroyed
    at GitRepository.module.exports.GitRepository.getRepo (/Applications/Atom.app/Contents/Resources/app.asar/src/git-repository.js:358:15)
    at GitRepository.module.exports.GitRepository.isPathIgnored (/Applications/Atom.app/Contents/Resources/app.asar/src/git-repository.js:224:19)
    at Repository.shouldExclude (/Users/Adam/.atom/packages/imdone-atom/lib/imdone-helper.coffee:18:30)
    at repo.watcher.chokidar.watch.ignored (/Users/Adam/.atom/packages/imdone-atom/node_modules/imdone-core/lib/mixins/repo-watched-fs-store.js:58:28)
    at testCriteria (/Users/Adam/.atom/packages/imdone-atom/node_modules/imdone-core/node_modules/chokidar/node_modules/anymatch/index.js:32:26)
    at Array.some (native)
    at anymatch (/Users/Adam/.atom/packages/imdone-atom/node_modules/imdone-core/node_modules/chokidar/node_modules/anymatch/index.js:59:48)
    at FSWatcher._isIgnored (/Users/Adam/.atom/packages/imdone-atom/node_modules/imdone-core/node_modules/chokidar/index.js:341:15)
    at FSWatcher.<anonymous> (/Users/Adam/.atom/packages/imdone-atom/node_modules/imdone-core/node_modules/chokidar/index.js:111:18)
    at FSWatcher.<anonymous> (/Users/Adam/.atom/packages/imdone-atom/node_modules/imdone-core/node_modules/chokidar/index.js:411:35)
    at /Users/Adam/.atom/packages/imdone-atom/node_modules/imdone-core/node_modules/chokidar/node_modules/readdirp/readdirp.js:214:72
    at Array.filter (native)
    at /Users/Adam/.atom/packages/imdone-atom/node_modules/imdone-core/node_modules/chokidar/node_modules/readdirp/readdirp.js:214:12
    at /Users/Adam/.atom/packages/imdone-atom/node_modules/imdone-core/node_modules/chokidar/node_modules/readdirp/readdirp.js:194:38
    at FSReqWrap.oncomplete (fs.js:82:15)
```
### Commands

```
  3x -3:22.2.0 tree-view:remove-project-folder (span.name.icon.icon-repo)
     -3:12.3.0 application:add-project-folder (div.header.list-item)
  4x -2:40.9.0 core:close (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
```
### Config

``` json
{
  "core": {
    "audioBeep": false,
    "themes": [
      "atom-material-ui",
      "atom-material-syntax"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
MagicPython, v0.5.7
antialiased, v0.1.0
atom-beautify, v0.28.19
atom-material-syntax, v0.2.10
atom-material-ui, v0.8.0
autoclose-html, v0.19.0
autocomplete-paths, v1.0.2
caniuse, v0.5.5
dash, v1.5.0
file-icons, v1.6.13
imdone-atom, v1.3.12
js-hyperclick, v1.4.1
linter, v1.11.3
linter-csslint, v1.1.0
minimap, v4.18.0
nav-panel, v0.0.11
neon-cursor, v1.4.0
open-in-browsers, v0.0.7
pigments, v0.19.3
tool-bar-main, v0.0.8
wakatime, v5.0.2

# Dev
No dev packages
```

**Comment by piascikj** [comment]
Are you still getting this error?

**Comment by shemerey** [comment]
@piascikj  I do

**Comment by piascikj** [comment]
Is anyone still seeing this?

**Comment by onetrev** [comment]
Definitely still seeing it.

**Comment by shemerey** [comment]
It happens after switch from project to porject by https://github.com/danielbrodin/atom-project-manager

**Comment by piascikj** [comment]
This should be fixed in 2.3.17.  Just published a patch.  I was able to reproduce after installing project manager.  @shemerey Thanks for the tip.  Let me know if it's working now.
